package.json

```diff
@@ -56,7 +56,7 @@
     "test-extra": "npm run test-api-log && npm run test-cache && npm run test-jobQueue && npm run test-view && npm run test-disablehttp && npm run test-cors && npm run test-limiter && npm run test-xfwd",
     "test-jobQueue": "npm run test-with-coverage -- --recursive tests/jobQueue",
     "test-limiter": "npm run test-with-coverage -- tests/limiter/limiter.js",
-    "test-main": "npm run test-with-coverage -- --recursive tests/api tests/clock tests/config tests/db tests/enforceToken tests/realtime tests/enableCache tests/logging",
+    "test-main": "npm run test-with-coverage -- --recursive tests/api tests/clock tests/config tests/db tests/enforceToken tests/realtime tests/enableCache tests/logging test/signal",
```

Comment:
```
tests/api tests/clock tests/config tests/db tests/enforceToken tests/realtime tests/enableCache tests/logging test/signal
```

should be in alphabetical order

```
tests/api tests/clock tests/config tests/db tests/enableCache tests/enforceToken tests/logging tests/realtime test/signal
```

And should `test/signal` be `tests/signal`?

Reply:
great catch! I had to move this file to the new script execution... process.exit is messing up - thankfully running as a new script it works (great learning, thx)